datamade/just-spaces
# Surveys submitted detail

## Overview

This PR implements a basic detail view with a side scrolling DataTable, following the pattern of the sample data in Gehl's [PLDP repo](https://github.com/gehl-institute/pldp/tree/master/Sample%20Data). It also includes merged bug fixes from #72 and #73.

## Testing Instructions

* If a `just-spaces` database already exists, drop it:  `dropdb just-spaces`
* Recreate it: `pg_restore -C -j4 --no-owner just-spaces.dump | psql`
* Run the server: python manage.py runserver
* Login is as testuser (pw in DataMade Lastpass)
* Click "Run survey" in the sidebar. Submit some data!
* Click "View collected data" in the sidebar and select the survey you just ran.
* Check if the data you just submitted shows up!

Connects #61
